Mentorship round2 (#1272)
nf-core/nf-co.re: markdown/mentorships.md

  - Writing an extensive tutorial about some aspect of nf-core

The exact scope and aims of each mentoring partnership can vary according to the participants' needs and situation.

> These will be judged on a case-by-case basis as part of the application process. You can find out more about what mentor and mentee pairs achieved in the first round of the program [here](https://www.nextflow.io/blog/2022/czi-mentorship-round-1.html).

Reviewer: ```suggestion
These will be judged on a case-by-case basis as part of the application process. You can find out more about what mentor and mentee pairs achieved in the first round of the program [in this Nextflow blog post](https://www.nextflow.io/blog/2022/czi-mentorship-round-1.html).
```

## Round 1 (Closed)

> Round 1 is now closed. Lots of interesting work has been done during this first edition of the mentorship program! Thank you to all participants! You can check the [Nextflow blog post]() for an overview and testimonials of this round.

Reviewer: ```suggestion
Round 1 is now closed. Lots of interesting work has been done during this first edition of the mentorship program! Thank you to all participants! You can check the [Nextflow blog post](https://www.nextflow.io/blog/2022/czi-mentorship-round-1.html) for an overview and testimonials of this round.
```

### Application process

Applications for Round 2 are now open!

<!-- _Applications are now closed. Those who have applied will hear from us soon. Thank you!_ -->

> Please submit an application using the forms below before November 1 to be considered for the program.

Reviewer: ```suggestion
Please submit an application using the forms below before November 1<sup>st</sup> to be considered for the program.
```
